### ISSUE TITLE
Add More language files

### ISSUE THREAD

**Issue by ivandiazwm** [ivandiazwm/phpback#1]
Hi there!
I recently added a CodeIgniters' multi language support to this project.

It is necessary to add more language files in `application/languages` folder.
Feel free to create a pull request adding more languages files.

Thank you!

**Comment by ramirovjr** [comment]
I created a portuguese (Brazil) translation for the project and submitted a PR.

**Comment by Jessar1994** [comment]
Is it just to translate the files in the language folder?

Or how can i translate phpback to Swedish?

**Comment by ivandiazwm** [comment]
Yes it is, just create a new folder with the language name. Then you can select which one you want from the admin panel.

Take this as an example:
28424f66e1e5fe703679a737834bca130fd7acc0

Warning:
1) The admin panel doesn't get translated. This is going to be added in translate files in the future.
2)Welcome message cannot be changed. The possibility of doing it is going to be able in v1.3.

**Comment by MarieCocci** [comment]
French in progress ...You should receive a pull request in the next few days.

Sorry for the several click on "flag comment" on your website... it was to understand what it was...
Maybe you could open a responsive window with a short message to explain something has been send to admin...

**Comment by thijshakkenberg** [comment]
Dutch done https://github.com/ivandiazwm/phpback/issues/69

**Comment by marcelklehr** [comment]
I can recommend https://translatewiki.net for leveraging the power of community ;)
